src/storage/db/rocksdb.rs

```diff
@@ -366,7 +366,7 @@ impl RocksDB {
             start_iterator_prefix
         } else {
             if let Some(page_token) = &page_options.page_token {
-                increment_vec_u8(&page_token)
+                [page_token.clone(), vec![0u8]].concat()
```

Comment:
Could you please add a comment with an example subkey? This might be hard to follow for someone without context.

Reply:
Done